### PR TITLE
remove top_cities

### LINKED ISSUE #322: Text Search from Home Page is Broken [Ceder-14]
The auto complete, and search functionality seems to be broken on the ceder-14 production. This should be a simple fix. In addition the popular cities do not seem to show any results when you click on them. 

screen shot 2017-03-27 at 11 16 46 am
screen shot 2017-03-27 at 11 16 33 am

### PR DESCRIPTION
**Pull request by mi-wood**
# Context
- Fixes #322 
- Removes top cities from landing page

# Summary of Changes

- remove top_cities from landing page

# Checklist

- [ ] Tested Mobile Responsiveness
- [ ] Added Unit Tests
- [ ] CI Passes
- [x] Deploys to Heroku on test Correctly (Maintainers will handle)
- [ ] Added Documentation (Service and Code when required)

### CODE REVIEW

**Comment by mi-wood** [comment]
Before:
screen shot 2017-04-04 at 8 38 07 pm
After:
screen shot 2017-04-04 at 8 38 14 pm

**Comment by mi-wood** [comment]
@tkwidmer kept the model method in there because it seems to be using it in some miscellaneous tests. I can circle back around and fix those if need be.

**Comment by tkwidmer** [comment]
deployed to pull request branch on Heroku.  Basic functionality seems to be ok. 

https://refuge-staging-14-pr-327.herokuapp.com

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -14,7 +14,6 @@
  *= require nav
  *= require search
  *= require _mobile
- *= require top_cities
```
Comment (tkwidmer) [inline app/assets/stylesheets/application.scss]: do we want to actually delete the actual file?

Comment (tkwidmer) [inline app/assets/stylesheets/application.scss]: https://github.com/RefugeRestrooms/refugerestrooms/blob/develop/app/assets/stylesheets/top_cities.scss

Comment (mi-wood) [inline app/assets/stylesheets/application.scss]: yeah, must have missed my `grep` :)

**Pull request merged**